scripts/1-process/dbGenerator/insertMDNMethodArgsDocumentation.ts

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env ts-node
+
+import * as Path from 'path';
+import db from '../../../db';
+import config from '../../../config';
+import DocumentationExtractor, { IMDNMethodParameter } from '../../../src/DocumentationExtractor';
+
+const interchangeableParamNames = new Map<string, string>([
```

Comment:
This is the hackiest thing I did. Basically was trying to find some way to whittle down the list of different params.